src/index.js

```diff
@@ -0,0 +1,142 @@
+const state = {
+  fahrenheit: 60,
+  city: 'Edmonds',
+  sky: 'Sunny',
+};
+let num = state.fahrenheit;
+const tempUp = (event) => {
+  state.fahrenheit += 1;
+  tempChangeColor(state.fahrenheit);
+  weatherGarden();
+  const increase = document.getElementById('f');
+  increase.textContent = `Degrees Fahrenheit:${state.fahrenheit}`;
+  return increase.textContent;
+};
+
+const tempDown = (event) => {
+  state.fahrenheit--;
+  tempChangeColor(state.fahrenheit);
+  weatherGarden();
+  const decrease = document.getElementById('f');
+  decrease.textContent = `Degrees Fahrenheit:${state.fahrenheit}`;
+  return decrease.textContent;
+};
+
+const cityInput = (event) => {
+  let cityInput = document.getElementById('input').value;
+  state.city = cityInput;
+  let cityHeader = document.getElementById('city');
+  cityHeader.textContent = `${state.city}`;
+  getCityWeather();
+};
+
+const tempChangeColor = (num) => {
+  let element = document.getElementById('temperatureContainer');
+  if (state.fahrenheit >= 80) {
+    element.style.backgroundColor = 'red';
+  } else if (80 > state.fahrenheit && state.fahrenheit >= 70) {
+    element.style.backgroundColor = 'orange';
+  } else if (70 > state.fahrenheit && state.fahrenheit >= 60) {
+    element.style.backgroundColor = 'yellow';
+  } else if (60 > state.fahrenheit && state.fahrenheit >= 50) {
+    element.style.backgroundColor = 'green';
+  } else if (50 > state.fahrenheit && state.fahrenheit >= 33) {
+    element.style.backgroundColor = 'teal';
+  } else if (state.fahrenheit <= 32) {
+    element.style.backgroundColor = 'blue';
+  }
+};
+
+const weatherGarden = () => {
```

Comment:
Consider renaming the method name to something a little more descriptive like `setWeatherGarden` or `updateWeatherGarden`